File: cpp/Q4.cpp
#include <iostream>

using namespace std;

int main(){
    //lembrar de deixar como float
    int num1, num2;
    float mediaHarmonica;
    cout << "Digite dois numeros: " << endl;
    cin >> num1 >> num2;
    mediaHarmonica = 2.0/((1.0/num1) + (1.0/num2));
    cout << mediaHarmonica << endl;
    
    return 0;
}

File: java/Q4.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
       Scanner scan = new Scanner(System.in);
       int num1, num2;
       double media;
       System.out.println("Digite dois numeros: ");
       num1 = scan.nextInt();
       num2 = scan.nextInt();
       media = 2.0/((1.0/num1)+(1.0/num2));
       System.out.println("A media harmonica é: "+media);
    }
}
